README.md

```diff
@@ -63,7 +63,8 @@ sudo pacman -S php php-fpm php-apcu nginx git
 Clone LibreY:
 
 ```sh
-mkdir -p /var/www/html
+sudo mkdir -p /var/www/html/LibreY
+sudo chown $(whoami) /var/html/www/LibreY
```

Comment:
leading on from the same idea, probably isn't the best practice to have /var/html/www/LibreY owned by whatever user happens to be configuring the system.